packages/boundless-popover/index.js

```diff
@@ -411,16 +417,34 @@ export default class Popover extends React.PureComponent {
         this.$caret.style[longitudinal ? 'top' : 'left'] = '0px';
     }
 
+    matchAnchorWidth = () => {
+      if (!this.props.matchAnchorWidth) return;
+
+      const anchor = this.props.anchor instanceof HTMLElement
+                     ? this.props.anchor
+                     : findDOMNode(this.props.anchor);
+      const dialog = this.dialog.$dialog;
+      const anchorWidth = anchor.getBoundingClientRect().width;
```

Comment:
We compute this in `cacheViewportCartography()` as called by `this.align()`, so if we move the invocation of this function to after the caching is complete, we can avoid doing the measurement work twice :smile: